src/main/java/com/hrznstudio/galacticraft/GalacticraftClient.java

```diff
@@ -72,17 +71,13 @@
 @Environment(EnvType.CLIENT)
 public class GalacticraftClient implements ClientModInitializer {
 
-    public static JsonCapes jsonCapes;
-    public static CapeLoader capeLoader;
+    public static CapesLoader capesLoader;
```

Comment:
Can this be a constant?
(Invoke `CapesLoader#load` in `#onInitializeClient` and not in the constructor)

Reply:
yeah sure

Reply:
resolved in 391c60c